README.md

```diff
@@ -38,6 +38,18 @@ $ npx styleguidist-visual test --help
 $ npx styleguidist-visual approve --help
 ```
 
+### Action States
+
+You can capture screenshots after simulating an action, by providing a selector and list of actions as props to the component wrapper like this:
+
+```
+  ```js { "props": { "data-action-selector": ".my-button", "data-action-states": "base,hover,focus,mouseDown" } }
```

Comment:
by splitting these into separate data-attributes we're limiting ourselves to taking action on the single selector, so we couldn't support "click on  the dropdown [screenshot], hover over an item [screenshot]...". maybe this should be a single attribute which accepts stringified JSON in a structure that makes it easier to extend later?

Reply:
Yeah, I think that's a good idea.  I'll try it.

Reply:
Ok. I think these are our two best options:

Use single quotes, which are not valid json, and convert them before parsing. 
```
```js { "props": { "data-description": "regular", "data-action-states": "[{'action':'base'},{'action':'hover','selector':'.y-button'},{'action':'focus','selector':'.y-button'},{'action':'mouseDown','selector':'.y-button'}]" } }
 ```

OR escaped double quotes:
```
```js { "props": { "data-description": "regular", "data-action-states": "[{\"action\":\"base\"},{\"action\":\"hover\",\"selector\":\".y-button\"},{\"action\":\"focus\",\"selector\":\".y-button\"},{\"action\":\"mouseDown\",\"selector\":\".y-button\"}]" } }
```

I'm leaning towards the escaped double quotes. @swese44 @unindented do you have an opinion?

Reply:
I've pushed up a change to use escaped double quotes.